samples/boards/stm32/usbc/sink/prj.conf

```diff
@@ -0,0 +1,5 @@
+CONFIG_USBC_TCPC=y
+CONFIG_USBC_TCPC_STM32=y
```

Comment:
This is not necessary, because USBC_TCPC_STM32 is by default y if a compatible node is defined.

Reply:
Done

Reply:
not addressed